app/components/ransack/hidden_field_component.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module Ransack
+  # Component to render the hidden search field within the ransack search form
```

Comment:
**nitpick (blocking):** Could you rename this to `HiddenSortFieldComponent` and update the comment to mention this is for `sort field` not `search field`

Reply:
Fixed in 48af86ae30b3a0a4640bf9c11b95e6d3a6c46b7b